### PR TITLE
Multi-row grid example

### LINKED ISSUE #14: [@thumbtack/tp-ui-react-grid]: Clarify how to make multi-row grids
Based on the Grid documentation it's not clear how to make multi-row grids using the React component.

Should either document our recommendation or change the API to make the component easier to use.

### PR DESCRIPTION
**Pull request by tomgenoni**
pixelsnap 2019-02-22 at 15 54 53
pixelsnap 2019-02-22 at 15 55 52

Fixes #14

### CODE REVIEW

**Comment by tomgenoni** [comment]
Don't love the wording but I think the example makes is clear.

File: www/src/pages/atomic/index.mdx [hunk 1]
```diff
@@ -458,6 +458,21 @@ These classes should be used in place of the [SCSS `grid`](/components/grid/scss
 </div>
 ```
 
+### Multi-row
+
+Using multiple columns with a combined width that exceed `12` will automatically wrap into new rows.
```
Comment (danoc) [inline www/src/pages/atomic/index.mdx]:
```suggestion
Using multiple columns with a combined width that exceeds `12` will automatically wrap into new rows.
```

File: www/src/pages/components/grid/react/index.mdx [hunk 2]
```diff
@@ -58,6 +58,33 @@ import { ComponentHeader, ComponentFooter } from 'components/thumbprint-componen
 </Grid>
 ```
 
+### Multi-row
+
+Using multiple columns with a combined width that exceed `12` will automatically wrap into new rows.
```
Comment (danoc) [inline www/src/pages/components/grid/react/index.mdx]:
```suggestion
Using multiple columns with a combined width that exceeds `12` will automatically wrap into new rows.
```

**Comment by tomgenoni** [comment]
I will update the example to be responsive. E.g., you might want:

mobile:

```
| a | b |
| c | d |
| e | f |
```

desktop:

```
| a | b | c |
| d | e | f |
```

**Pull request merged**